site/docs/v1/tech/apis/_tenant-request-body.adoc

```diff
@@ -442,7 +489,7 @@ include::../shared/_premium-edition-blurb-api.adoc[]
 :premium_feature!:
 
 [field]#tenant.usernameConfiguration.unique.numberOfDigits# [type]#[Integer]# [optional]#Optional# [default]#defaults to `5`# [since]#Available since 1.27.0#::
-The maximum number of digits to use when building a unique suffix for a username.  A number will be randomly selected and will be 1 or more digits up to this configured value in length. For example, if this value is `5`, the suffix will be a number between `00001` and `99999`, inclusive and left padded with zeros. The value of this field must be greater than or equal to `3` and less than or equal to `10`.
+The maximum number of digits to use when building a unique suffix for a username. A number will be randomly selected and will be 1 or more digits up to this configured value in length. For example, if this value is `5`, the suffix will be a number between `00001` and `99999`, inclusive. The value of this field must be greater than or equal to `3` and less than or equal to `10`.
```

Comment:
Did you double check that? I seem to remember testing and seeing the left padding with zeros.

Reply:
Based on my testing, I am not seeing that it is left padding with zeros.  I am seeing configurations like `johnny123#25874728` but none of my API Tested users are coming back with left padded zeros

Reply:
The padding is random. it could be `1` or `01`, or `001` or something like that.